hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/ActionsITest.groovy

```diff
@@ -333,9 +333,7 @@ class ActionsITest extends AbstractITestBase {
         assertEquals(1, resp.data.size())
 
         // ENABLE Trigger
-        testTrigger.setEnabled(true);
-
-        resp = client.put(path: "triggers/test-email-threshold", body: testTrigger)
+        resp = client.put(path: "triggers/enabled", query:[triggerIds:"test-email-threshold",enabled:true] )
```

Comment:
I'm not sure I understand the regression risk?  The point of the new endpoints is to reduce risk (and add convenience) when just performing an enable or disable.  I have updated all of these in the existing tests.  I can go back and undo those changes but can you better explain why that is necessary?